packages/styles/src/components/card.scss

```diff
@@ -10,26 +10,79 @@
 @use './../mixins/icons' as icons-mx;
 @use './../variables/icons';
 @use './../variables/animation';
+@use '../mixins/utilities' as utilities-mx;
 
 @use './../functions/tokens';
 @use './../tokens/utilities';
+@use './../tokens/components';
 
-tokens.$default-map: utilities.$post-spacing;
+tokens.$default-map: components.$post-cards;
 
 .card-group {
   margin-top: commons.$border-width;
   margin-left: commons.$border-width;
-  box-shadow: commons.$box-shadow-lg;
+  box-shadow: tokens.get('card-interactive-elevation');
+  border-radius: tokens.get('card-border-radius');
 }
 
 .card {
-  transition: box-shadow 0.25s;
-  border: 0;
-  border-radius: 0;
-  // Outline with card-group margin trick prevents double/jumping (on hover) borders
-  outline: 1px solid card.$card-border-color;
-  box-shadow: commons.$box-shadow-lg;
+  transition: all 250ms;
   color: card.$card-color;
+  border-radius: tokens.get('card-border-radius');
+  background-color: tokens.get('card-enabled-bg1');
+
+  .palette-alternate & {
+    background-color: tokens.get('card-enabled-bg2');
+  }
+
+  // Add an elevation only if card is interactive
+  &.interactive-card {
```

Comment:
It seems that the interactive cards are the same as cards wrapped in a post-linkarea: https://next.design-system.post.ch/?path=/docs/1d52b794-768b-464e-90eb-4fd15774aa90--docs

As the post-linkarea is more flexible and potentially better for accessibility, I would drop the `.interactive-card` class and set the styles on the `post-likarea .card`.